lib/sdf-server/src/service/v2/management/generate_template.rs

```diff
@@ -71,42 +80,107 @@ pub async fn generate_template(
     PosthogClient(posthog_client): PosthogClient,
     OriginalUri(original_uri): OriginalUri,
     Host(host_name): Host,
-    Path((_workspace_pk, change_set_id, view_id)): Path<(WorkspacePk, ChangeSetId, ViewId)>,
```

Comment:
Since we're editing this endpoint, it's a good change to upgrade the extractors to use `ChangeSetDalContext`, like thus : https://github.com/systeminit/si/blob/63fdd1d46f2de7008a0ce24af190c2543ce7a3a7/lib/sdf-server/src/service/v2/view/paste_component.rs#L53-L59

This way you can get rid of most of these arguments (and also ignore the changeset in path, since that gets deal with by the extractor)